sass/base/space/_space.scss

```diff
@@ -5,46 +5,49 @@
   getThemeProperty(layoutSpace) variable is defined in _fw-variables.scss
   Used for consistent margins and gutters throughout the defined content
 */
+
+$default-base-space-properties:(
+  paddingclasses:(
+    default: getThemeProperty(layoutSpace),
+    double: getThemeProperty(layoutSpace) * 2,
+    triple: getThemeProperty(layoutSpace) * 3,
+    one-forth: getThemeProperty(layoutSpace) * 0.25,
+    half: math.div(getThemeProperty(layoutSpace), 2),
+    three-forths: getThemeProperty(layoutSpace) * 0.75,
+    noleft: inherit inherit inherit 0,
+    noright: inherit 0 inherit inherit,
+    halfleft: inherit inherit inherit math.div(getThemeProperty(layoutSpace), 2),
+    halfright: inherit math.div(getThemeProperty(layoutSpace), 2) inherit inherit,
+    notop: 0 inherit inherit inherit,
+    nobottom: inherit inherit 0 inherit,
+    halftop: math.div(getThemeProperty(layoutSpace), 2) inherit inherit inherit,
+    halfbottom: inherit inherit math.div(getThemeProperty(layoutSpace), 2) inherit,
+  )
+);
+// If $base-space-properties is set, lets merge into our defaults
+$base-space-properties: $default-base-space-properties !default;
+@if($base-space-properties){
+  $base-space-properties: recursive-map-merge($default-base-space-properties, $base-space-properties);
+}
+
+// Default spacing - desktop
 .fw-space,
 .space{
-  padding: getThemeProperty(layoutSpace);
-  &.double{
-    padding: getThemeProperty(layoutSpace) * 2;
-  }
-  &.triple{
-    padding: getThemeProperty(layoutSpace) * 3;
-  }
-  &.one-forth{
-    padding: getThemeProperty(layoutSpace) * 0.25;
-  }
-  &.half{
-    padding: math.div(getThemeProperty(layoutSpace), 2);
-  }
-  &.three-forths{
-    padding: getThemeProperty(layoutSpace) * 0.75;
-  }
-	&.noleft{
-		padding-left: 0;
-	}
-	&.noright{
-		padding-right: 0;
-	}
-	&.halfleft{
-		padding-left: math.div(getThemeProperty(layoutSpace), 2);
-	}
-	&.halfright{
-		padding-right: math.div(getThemeProperty(layoutSpace), 2);
+	padding: getThemeProperty(layoutSpace);
+	@each $label, $value in getThemeProperty("paddingclasses", $base-space-properties){
+		&.#{$label}{
+			padding: #{$value};
+		}
 	}
-	&.notop{
-		padding-top: 0;
-	}
-	&.nobottom{
-		padding-bottom: 0;
-	}
-	&.halftop{
-		padding-top: math.div(getThemeProperty(layoutSpace), 2);
-	}
-	&.halfbottom{
-		padding-bottom: math.div(getThemeProperty(layoutSpace), 2);
+}
+
+// For each responsive breakpoint, create a space class
+@each $label, $breakpoint in getThemeProperty(breakpointResponsive){
+	@media only screen and (max-width: $breakpoint) {
+		@each $spacelabel, $value in getThemeProperty("paddingclasses", $base-space-properties){
+			.fw-space-#{$label}-#{$spacelabel}{
```

Comment:
I think a good workaround to make this more specific is to include `.fw-space` in the selector:
```scss
.fw-space.fw-space-#{$label}-#{$spacelabel}{
```